https://issues.redhat.com/browse/ACM-16784 (#7409)
open-cluster-management/rhacm-docs: release_notes/errata.adoc

See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.

== Errata 2.10.7

* Delivers updates to one or more product container images.

* Fixes a regression that made `CertificatePolicy` compliance updates fail when the message was longer than 1024 characters. (link:https://issues.redhat.com/browse/ACM-14627[ACM-14627])

* Updates the `CertificatePolicy` status to use an expiration timestamp, which is more accurate since it is absolute. (link:https://issues.redhat.com/browse/ACM-13736[ACM-13736])

* Adds code and an underscore before the first digit for cluster labels that start with a digit and outputs to the metrics, `acm_managed_cluster_labels`. (link:https://issues.redhat.com/browse/ACM-14483[ACM-14483])

> * Updates the InfraEnv `osImageVersion` field so that it scales clusters that have been upgraded since the initial install. (link:https://issues.redhat.com/browse/ACM-14601[ACM-14601])

Reviewer: ```suggestion
* Updates the InfraEnv `osImageVersion` field so that it scales clusters that were upgraded since the initial install. (link:https://issues.redhat.com/browse/ACM-14601[ACM-14601])
```

> * Changes the system so that the network time protocol (NTP) server specified in the InfraEnv gets used as a fallback.(link:https://issues.redhat.com/browse/ACM-16166[ACM-16166])

Reviewer: ```suggestion
* Changes the system so that the network time protocol (NTP) server that is specified in the InfraEnv gets used as a fallback.(link:https://issues.redhat.com/browse/ACM-16166[ACM-16166])
```

> * Tracks performance and scale tests of the `multicluster-observability-operator` pod to monitor issues from earlier pod logs. (link:https://issues.redhat.com/browse/ACM-14586[ACM-14586])

Author: FYI, the last two bullets (line 24 and line 26) I made these summaries from reading through the Jiras and looking at the PRs that resolved the Jiras. I think they're good but let me know what you think!

Reviewer 2: lgtm! thanks for reviewing the jiras and resolved prs to summarize the fixes.
